src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,69 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onMovieAdd: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ onMovieAdd }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [query, setQuery] = useState('');
+
+  const DEFOULT_IMG
+    = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    setError(null);
+  };
+
+  const handleSubmit: React.FormEventHandler<HTMLFormElement> = (event) => {
+    event.preventDefault();
+
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`